dcos-monitoring: v1.0.0 service docs

gracedo opened the pull request:
## Description
We are preparing to release GA v1.0.0. These docs were automatically generated from the docs in the [dcos-montoring repo](https://github.com/mesosphere/dcos-monitoring/tree/master/docs). Most of this documentation has already passed reviews that occurred in the dcos-monitoring repo. Any updates to the docs coming out of this review should be made in the dcos-monitoring repo and re-committed here automatically. Do not make updates here.

We are on rc4 right now, and hopefully there should be no major updates to docs before the final release candidate. I'd like us to get started on reviewing the docs now so we can release faster when it's ready.

Linked docs ticket: https://jira.mesosphere.com/browse/DCOS-54543

## Urgency
- [ ] Blocker <!-- Ping @pavisandhu for review -->
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

anithamuthu79 commented on pages/services/dcos-monitoring/1.0.0/operations/grafana/dashboard-configs/index.md:
````diff
@@ -0,0 +1,149 @@
+dcos security secrets create -f <PATH_TO_PRIVATE_KEY> gitsshkey-secret
+```
+
+For Github, please make sure to add [Deployment Key](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) (i.e., the public key) to the repository.
````
Replace the following:
`Github` with `GitHub` 
`please` with `you must`

anithamuthu79 commented on pages/services/dcos-monitoring/1.0.0/operations/grafana/dashboard-configs/index.md:
````diff
@@ -0,0 +1,149 @@
+}
+```
+
+Note that you'll have to use `git@github.com:<USER>/<REPO>.git` instead of `https` as the scheme of the URL.
````
Use the `Note` message format listed on https://wiki.mesosphere.com/display/DOCS/Documentation+Style+Guide#DocumentationStyleGuide-Notes

anithamuthu79 commented on pages/services/dcos-monitoring/1.0.0/operations/grafana/index.md:
```diff
@@ -0,0 +1,11 @@
+model: ../../data.yml
+---
+
+This section walks you through how to configure the Grafana bundled with the {{ model.techName }} service.
```
Replace `This section` with `This page`.

anithamuthu79 commented on pages/services/dcos-monitoring/1.0.0/operations/grafana/ui/index.md:
```diff
@@ -0,0 +1,68 @@
+
+By default, the username and password is `admin:admin`. See the [Admin Credentials section](#admin-credentials) for directions on configuring these credentials.
+
+Note that you need to set `grafana.admin_router_proxy` to `false`.
```
Use `Note` message format listed on https://wiki.mesosphere.com/display/DOCS/Documentation+Style+Guide#DocumentationStyleGuide-Notes

anithamuthu79 commented on pages/services/dcos-monitoring/1.0.0/operations/prometheus/alert-rules/index.md:
````diff
@@ -0,0 +1,163 @@
+dcos security secrets create -f <PATH_TO_PRIVATE_KEY> gitsshkey-secret
+```
+
+For Github, please make sure to add [Deployment Key](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) (i.e., the public key) to the repository.
````
Replace the following:
`Github` with `GitHub` 
`please` with `you must`

anithamuthu79 commented on pages/services/dcos-monitoring/1.0.0/operations/prometheus/alert-rules/index.md:
````diff
@@ -0,0 +1,163 @@
+}
+```
+
+Note that you'll have to use `git@github.com:<USER>/<REPO>.git` instead of `https` as the scheme of the URL.
````
Use the `Note` message format as listed on https://wiki.mesosphere.com/display/DOCS/Documentation+Style+Guide#DocumentationStyleGuide-Notes

anithamuthu79 commented on pages/services/dcos-monitoring/1.0.0/operations/prometheus/index.md:
```diff
@@ -0,0 +1,11 @@
+model: ../../data.yml
+---
+
+This section explains how to configure the Prometheus bundled with the {{ model.techName }} service.
```
Replace `section` with `page`

anithamuthu79 commented on pages/services/dcos-monitoring/1.0.0/operations/prometheus/ui/index.md:
```diff
@@ -0,0 +1,59 @@
+
+You can also choose to access the Prometheus UI through [Marathon LB](https://docs.mesosphere.com/services/marathon-lb/1.12.x/) by configuring the service.
+
+Please configure `prometheus.marathon_lb_proxy.vhost` field to be the Fully Qualified Domain Name (FQDN) for the service.
```
Replace `Please` with ` You must`.

anithamuthu79 commented on pages/services/dcos-monitoring/1.0.0/operations/prometheus/ui/index.md:
````diff
@@ -0,0 +1,59 @@
+```
+
+<p class="message--note"><strong>NOTE:</strong> There is one limitation currently that the service name cannot contain any character that is not allowed in a DNS name (e.g., /).
+That means the service cannot be installed in a Marathon folder. </p>
````
Replace `<p class="message--note"><strong>NOTE:</strong> There is one limitation currently that the service name cannot contain any character that is not allowed in a DNS name (e.g., /).
That means the service cannot be installed in a Marathon folder. </p>`

with `<p class="message--note"><strong>NOTE:</strong> Currently, there is a limitation where the service name cannot contain any character that is not allowed in a DNS name (e.g., /), which means the service cannot be installed in a Marathon folder. </p>

gracedo commented:
@anithamuthu79 Addressed review comments [here](https://github.com/mesosphere/dcos-monitoring/pull/175). The next release candidate cut will force push those changes back to this branch.

gracedo commented:
Linked documentation ticket: https://jira.mesosphere.com/browse/DCOS-54543

gracedo commented:
@anithamuthu79 Please merge this in, we have released dcos-monitoring v1.0.0 (GA)
